bigbluebutton-html5/imports/ui/components/nav-bar/service.js

```diff
@@ -1,22 +1,8 @@
-import Auth from '/imports/ui/services/auth';
 import Breakouts from '/imports/api/breakouts';
 
-const getBreakouts = () => Breakouts.find().fetch();
-
-const getBreakoutJoinURL = (breakout) => {
-  const currentUserId = Auth.userID;
-
-  if (breakout.users) {
-    const user = breakout.users.find(u => u.userId === currentUserId);
-
-    if (user) {
-      return user.redirectToHtml5JoinURL;
-    }
-  }
-  return '';
-};
+const getBreakouts = () => Breakouts.find().fetch().sort((a, b) => a.sequence > b.sequence);
```

Comment:
I believe you can sort without `fetch()` -- https://docs.mongodb.com/manual/reference/method/cursor.sort/